docs/authentication/saml/account-linking.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Account Linking
+description: Learn how Clerk handles account linking during SAML SSO.
+---
+
+# Account Linking
+
+Account Linking is a process used by Clerk to ensure a smooth sign-in and sign-up experience using both SAML SSO and other methods (e.g. username/password). By using the "email address" as the common identifier, Clerk aims to merge accounts whenever possible. This mechanism is triggered when an SAML provider returns an email address that matches an existing account, assuming a single owner for each email address.
+
+## How it works
+
+When a user attempts to sign in or sign up, Clerk first checks the email address provided. Clerk will attempt to link the SAML account with any existing Clerk account that shares the same email address.
+
+In the following sections, we'll look at the different scenarios that can occur during this process and explain how Clerk handles each one.
+
+<Callout type="info">
+    Email addresses from SAML providers are considered verified by default!
+</Callout>
+
+<Images
+    width={2400}
+    height={1754}
+    src="/docs/images/authentication/account-linking-flow-saml.webp"
+    alt="Flow chart of the SAML SSO account linking process in various scenarios"
+/>
+
+### Email address is verified in Clerk
+
+In this scenario, when a user signs in using a SAML provider that returns a matching verified email address, Clerk links the SAML account to the existing account and signs the user in. This even applies to password-protected accounts, as the SAML sign-in process automatically bypasses password verification.
+
+### Email address is not verified in Clerk but instance doesn't require verification during Sign-up
```

Comment:
```suggestion
### Email is unverified and verification isn't required
```

Trying to shorten this up, as it looks big in the ToC
![image](https://github.com/clerk/clerk-docs/assets/5156915/4e91f796-ab1c-4736-a1a8-3ce54411dae1)